tests/ert/unit_tests/config/config_dict_generator.py

```diff
@@ -342,7 +342,7 @@ def to_config_dict(self, config_file, cwd, all_defines=True):
                     name,
                     (
                         fn,
-                        f"EXECUTABLE script/{name}.exe\nMIN_ARG 0\nMAX_ARG 1\n",
+                        open(fn, encoding="utf-8").read(),  # noqa SIM115
```

Comment:
You can use `pathlib`'s `read_text()` function to avoid adding the ruff exception.

Reply:
`{name}.exe` was never correct. In the failing instance hypothesis generated two jobs with different name that used the job config. The EXECUTABLE should in that case be the same.

When and why hypothesis decided to generate this lately I do not know